src/core/Link/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { useStyles } from './styles';
+import { BaseLinkProps } from './base';
+
+interface LinksProps extends BaseLinkProps {
+  pathname: string;
```

Comment:
using camelCase for `pathname` would be better

Reply:
I think `pathname` is better. Link/history uses the same format to specify the path.